app/controllers/foods_controller.rb

```diff
@@ -1,3 +1,33 @@
 class FoodsController < ApplicationController
-  def index; end
+  def new
+    @user = current_user
```

Comment:
**OPTIONAL** Since you're not using this variable, it would be good to remove this line to keep the code clean.